src/lavinmq/client/channel.cr

```diff
@@ -358,7 +358,13 @@ module LavinMQ
           unless frame.no_wait
             send AMQP::Frame::Basic::ConsumeOk.new(frame.channel, frame.consumer_tag)
           end
-          c = Consumer.new(self, frame.consumer_tag, q, frame.no_ack, frame.exclusive, priority)
+
+          c = if q.is_stream_queue?
```

Comment:
Yeah, that looks better. But class is Queue here for a stream. I guess I need to fix that somewhere else? (Because @queues on vhost is `Hash(String, Queue)` ?)